Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -70,16 +70,38 @@ public static void UnhookEvents()
         public static void VbeEventCallback(IntPtr hWinEventHook, uint eventType, IntPtr hwnd, int idObject, int idChild,
             uint dwEventThread, uint dwmsEventTime)
         {
+            if (hwnd == IntPtr.Zero) { return; }
             //This is an output window firehose, leave this here, but comment it out when done.
-            //if (idObject != (int)ObjId.Cursor) Debug.WriteLine("Hwnd: {0:X4} - EventType {1:X4}, idObject {2}, idChild {3}", (int)hwnd, eventType, idObject, idChild);
+            if (idObject != (int)ObjId.Cursor)
+            {
+                Debug.WriteLine("Hwnd: {0:X4} - EventType {1:X4}, idObject {2}, idChild {3}", (int)hwnd, eventType, idObject, idChild);
```

Comment:
so why is this commented in again?

Reply:
Because we don't want to leave it on?

Reply:
But why keep the `Debug.WriteLine`? We should have that block commented out as per the comment, no?

Reply:
@bclothier oh. well yeah :smile:

Reply:
@Vogel612 sorry I somehow misread "commented in"